components/MadeWithPrezly/MadeWithPrezly.tsx

```diff
@@ -0,0 +1,43 @@
+import classNames from 'classnames';
+import { useEffect, useState } from 'react';
+
+import { LogoPrezly } from 'icons';
+
+import styles from './MadeWithPrezly.module.scss';
+
+const FOOTER_HEIGHT = 88;
+
+export function MadeWithPrezly() {
+    const [isMadeWithPrezlyVisible, setIsMadeWithPrezlyVisible] = useState(true);
```

Comment:
Since this state is encapsulated in the component, you can use shorter names like `isVisible` :)